README.md

```diff
@@ -2,8 +2,11 @@ Cheers
 =======
 
 ### Setting up the database
-Add a file `.env.local` to the root folder and configure `DATABASE_URL` to match your environment.
-Leave the database name out of the URL; rails will fill it according to the database.yml config.
+Add a file .env.local to the root folder and configure DATABASE_URL to
+match your environment (default: "DATABASE_URL:
+postgresql://localhost:5432). Leave the database name out of the URL;
+rails will fill it according to the database.yml config.Must have
```

Comment:
nitpick: we should probably have a space between "config." and "Must" in this paragraph.